src/components/ClawKeepApp.tsx

```
@@ -601,7 +588,7 @@ export default function ClawKeepApp() {
                 busyKind={
                   busy === "restore"
                     ? "restore"
-                    : busy === "backup" || isBackupRunning(status)
+                    : busy === "backup" || isBackupRunning(status, Date.now())
```

ESLint

[error] Error: Cannot call impure function during render

`Date.now` is an impure function. Calling an impure function can produce unstable results that update unpredictably when the component happens to re-render. (https://react.dev/reference/rules/components-and-hooks-must-be-pure#components-and-hooks-must-be-idempotent).

/home/jailuser/git/src/components/ClawKeepApp.tsx:591:68
  589 |                   busy === "restore"
  590 |                     ? "restore"
> 591 |                     : busy === "backup" || isBackupRunning(status, Date.now())
      |                                                                    ^^^^^^^^^^ Cannot call impure function
  592 |                     ? "backup"
  593 |                     : null
  594 |                 }

(react-hooks/purity)